data/human/culture conversations.txt

```diff
@@ -685,3 +685,70 @@ mission "Earth Day Blackout"
+			`	There's a counter in the corner with two levels, apparently so that the Burthenian attending it can serve both offwolders and locals. Alongside the typical drinks you've seen in spaceport bars and some local to the southern Core, the counter also offers a local drink called the "Antigrav" that claims to "ease visitors from Burthen's natural pull", and comes in a specially made glass, all for 30 credits.`
+			branch "no credits"
+				credits < 2
+			choice
+				`	(Order a regular drink from the counter.)`
+				`	(Order the Antigrav from the counter.)`
+					goto antigrav
+					to display
+						credits >= 30
+				`	(Don't order anything and leave the tent.)`
+					goto "leave tent"
+
+			action
+				payment -2
+			`	You hand over a couple credits for the drink and leave the tent, sipping with some difficulty as you go. The Burthensider at the counter advises you not to drink too much of it at once, and not to drop the cup "unless you want a munted foot for a few weeks."`
+				goto "leave tent"
+
+			label antigrav
+			action
+				payment -30
+				outfit "Antigrav Glass"
+			`	It turns out the glass the Antigrav comes with has a miniature repulsor installed so that you don't need to hold it while drinking it. The drink itself is a highly carbonated soda that tastes like something in between lingonberry and raspberry, made from the native berries found in Westland, the district the spaceport is located in. The Burthenian at the counter advises you not to drink too much of it at once and to not turn it off while it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`
+				goto "leave tent"
+
+			label "no credits"
+			`	Unfortunately, it looks like all the drinks are too pricy for you.`
+			choice
+				`	(Leave the tent.)`
+
+			label "leave tent"
+			`	The crowd is a bit more active when you return outside, and over the heads of the locals you see on a large, portable screen that the leading triathlete is a particularly stocky Burthenian, wearing a white helmet and tight, dark sportswear.`
+				goto "after tent"
+
+			label outside
+			`	Staying outside in the oppressive gravity and the chilly temperature, you move over to the sidelines and watch the large, portable screen that is broadcasting the leading competitors from above. It appears that competitors are divided into Burthenians and offworlders, with Burthenians wearing white helmets and offworlders wearing red helmets. It's nearing the end of the whole event; the competitor in front is named Abernathy, a stocky Burthenian apparently doing his eleventh Burthen Triathlon. Meanwhile, the leading offworlder is five hundred meters behind him, a competitor from Martini named Caldarone. Apparently, they've been at it for over 11 hours, beginning with a 1.5 kilometer swim, then a 40 kilometer bicycle ride, and finally a 10 kilometer run.`
```

Comment:
Is 11 hours realistic? Current Olympic (Earth) times for the same event and distances are about three hours. Even factoring in Burthen's heavier gravity, I would still expect the native-born Burthenian world champion to be in the three to five hour range.

Reply:
Perhaps it isn't.

Originally the competition was going to be more like NZ's Coast to Coast, which is 243km of running, kayaking and biking. The record for that is 10:34:37, according to Wikipedia.